pxf/pxf-service/src/main/resources/pxf-profiles-default.xml

```diff
@@ -128,4 +128,26 @@ under the License.
             <resolver>org.apache.hawq.pxf.plugins.gemfirexd.GemFireXDResolver</resolver>
         </plugins>
     </profile>
+	<profile>
+		<name>JSON</name>
+		<description>A profile for JSON data, one JSON record per line
+		</description>
+		<plugins>
+			<fragmenter>org.apache.hawq.pxf.plugins.hdfs.HdfsDataFragmenter</fragmenter>
+			<accessor>org.apache.hawq.pxf.plugins.json.JsonAccessor</accessor>
+			<resolver>org.apache.hawq.pxf.plugins.json.JsonResolver</resolver>
+		</plugins>
+	</profile>
+	<profile>
+		<name>JSON-PP</name>
```

Comment:
JSON-PP differs from JSON only by the hard coded property `onerecordperline=false`.  
I will remove it though in favor of JSON + explicit parameter in the DDL definition.
About the name shall i change it from `JSON` to `Json`?